elo_boat.py

```diff
@@ -55,6 +51,18 @@
 trueenv.make_as_global()
 
 
+def is_mention(text):
+    if "<@" in text and "<@&" not in text and any(i.isdigit() for i in text):
+        return True
+    return False
+
+
+def is_warcraft_name(text):
+    if "#" in text and "@" not in text and any(i.isdigit() for i in text):
+        return True
+    return False

```

Comment:
From what i see it doesn't check if it's a warcraft name, it's only check if the syntax is battletag-compatible.

I suggest to change name : is_warcraft_name => is_btag_formatted

Reply:
changed it to is_warcraft_name_format